### DISCUSSION TITLE
Bisq2 Easy : "half escrow" for the seller

### DISCUSSION THREAD

**Discussion by mdedonno1337** [bisq-network/bisq2#2992] [Ideas]
Hi,

Idea regarding the Bisq Easy protocol, in particular to reduce needs to trust the reputation:

How about adding an escrow (1-of-2 (seller and bisq) or with a timelock) to ensure the btc seller has locked the value in case of a dispute ?

I understand that the Easy protocol has been developed with new users that dont have btc to lock in an escrow, like for Bisq1, but I do not understand how Bisq2 ensures that the buyer will have the BTC in case of a dispute.
To ask it otherwise, if there is some scammers, how can we increase the cost only for the scammers and ensuring that new users can buy, and honnest people are still willing to use Bisq ?

The mandatory vendor-only escrow will not be an issue for honest vendors since they already have BTC that can be locked in an escrow.

**Comment by HenrikJannsen** [comment]
> How about adding an escrow (1-of-2 (seller and bisq) or with a timelock) 

You mean a 1of2 Multisig deposit for the seller? That would bring the problem about having an escrow as part of the trade which we try to avoid as it would add trust issues. Also a 1of2 would not secure it as the seller could always transfer themself.

The security in Bisq Easy is based on the sellers reputation. With the recent version the trade amount is also linked to that reputation level, thus a seller with low reputation cannot trade higher amounts (only if the buyer ignores the warnings and take the sellers offer with a too high amount that could happen, but then the buyer was made aware of the risks).

To get sufficient reputation the seller need to either burn BSQ or bond BSQ. Both are a substantial financial commitment and in both cases a scamming seller would render those commitments invalid as he would get banned from the network and the reputation is linked to his user profile. Thus creating a new fresh user profile will not help him to get back his reputation.
